src/main/java/frc/robot/subsystems/CargoIntake.java

```diff
@@ -63,16 +69,25 @@ public void stopRollers() {
   }
 
   /* TODO: check when limit switches return true */
-  public void spinRollersInWithLimits(){
-    if (!getRollerLimitState()) {
+  public void spinRollersInWithSensor() {
+    if (!isHoldingBall()) {
       spinRollersIn();
     }
   }
 
-  public boolean getRollerLimitState() {
-    return m_rollerSwitch.get();
+  public int getRollerIR() {
+    return m_rollerIR.getValue();
+  }
+
+  /* TODO: change the constant so it matches a real value */
+  public boolean isHoldingBall() {
```

Comment:
Rename to "isHoldingCargo" or "isCargoPossessed" for consistency - since this is the "CargoIntake" subsystem